app/main/views/briefs.py

```diff
@@ -116,6 +116,11 @@ def list_briefs():
 
     page = get_valid_page_or_1()
 
+    if request.args.get('with_users') == 'true':
```

Comment:
This could be shortened to:

`with_users = (request.args.get('with_users') == 'true')`